### ISSUE TITLE
Some confusion in quant_ops.py

### ISSUE THREAD

**Issue by xxktju** [colorjam/PAMS#5]
Thank you for sharing your excellent work. 
I am confusied about the line 78 in quant_ops.py, why the ‘self.epoch’ was always set to 1, shouldn't it change over the course of training？
Another confusion is the line 94 and line 97 in quant_ops.py, ‘’self.training‘’ seemed not be defined？

**Comment by colorjam** [comment]
Hi, @xxktju, we do ema at the first epoch, so initialize `self.epoch` as 1. `self.epoch` will increase during training: https://github.com/colorjam/PAMS/blob/593a4c7b0d46bd92f56ff494ae85ba2b4f97464d/main.py#L74

`self.training` is the attribute inherit from `nn.Module`, which represents the module is in training or evaluation mode.

**Comment by xxktju** [comment]
Thanks very much. Thank you for sharing your excellent work again.
